docs/README.md

```diff
@@ -1,188 +1,369 @@
+0.002929680     43207314        0.96716 3870319 C       361194  0.002694540     22:43207314:CCAAA:C     22_43603320_CCAAA_C     1.0873  0.55765 CCAAA   22
+-0.000095110    123260687       0.75512 3850019 T       361194  0.001116510     10:123260687:C:T        rs17587364      -0.0852 0.03053 C       10
+-0.000808090    4550458 0.77164 3850019 C       361194  0.001143620     2:4550458:A:C   rs1351242       -0.7066 0.31893 A       2
+-0.000689003    113936998       0.46018 3850019 T       361194  0.000961501     9:113936998:C:T rs1888161       -0.7166 0.32456 C       9
+0.003135670     202384532       0.96145 3850019 T       361194  0.002567760     2:202384532:C:T rs552928178     1.2212  0.65360 C       2
 
 ```
-gwasstudio --help
+</details>
 
-Usage: gwasstudio [OPTIONS] COMMAND [ARGS]...
+<br>
 
-  GWASStudio
+To harmonize and format data in the appropriate way please refer to this other repo (https://github.com/ht-diva/harmonization_pipeline)
 
-Dask options:
-  --distribute                  Distribute the load to a Dask cluster
-  --minimum_workers INTEGER     Minimum amount of running workers
-  --maximum_workers INTEGER     Maximum amount of running workers
-  --memory_workers TEXT         Memory amount per worker
-  --cpu_workers INTEGER         CPU numbers per worker
+### 1.1.2 Metadata preparation
 
-TileDB configuration:
-  --aws-access-key-id TEXT      aws access key id
-  --aws-secret-access-key TEXT  aws access key
-  --aws-endpoint-override TEXT  endpoint where to connect
-  --aws-use-virtual-addressing TEXT
-                                virtual address option
-  --aws-scheme TEXT             type of scheme used at the endpoint
-  --aws-region TEXT             region where the s3 bucket is located
-  --aws-verify-ssl TEXT         if ssl verification is needed
+The ingestion of summary statistics data need always to be accompanyed by its relative metadata. Below are the mandatory and optional fields of the metadata table together with an example of metadata table
+
+Mandatory columns
+
+| Column | Description |
+| --- | --- |
+| `project` | Project to which the data belongs (e.g. UKB, GTEx, UKB-PPP, etc...) |
+| `study` | The study under which the summary statistics fall. Example (a, b, d, which are groups of UK Biobank studies withing Open GWAS) |
+| `file_path` | The path where the files are stored |
+| `category` | The type of summary staitsics to be stored (GWAS, QTL) |
+
+Optional columns
+
+| Column | Description |
+| --- | --- |
+| `notes_sex` | The sex of the individuals that participated to that study (e.g., Males and Females)|
+| `population` | The ancestry of the population used for the study |
+| `total_samples` | Total sample size |
+| `total_cases` | Total amount of cases for binary traits|
+| `total_controls` | Total amount of control for binary traits |
+| `trait_desc` | A description of the summary statistics study |
+| `build` | Builds of the summary statistics (e.g., hg38, hg19, GRCh38, GRCh37, etc....) |
```

Comment:
The only allowable values are: https://github.com/ht-diva/gwasstudio/blob/8e1b796daa1b2e4541705656f322b6b1088a5ef3/src/gwasstudio/config/config.yaml#L31